dhcp/unikernel.ml

```diff
@@ -0,0 +1,85 @@
+open V1_LWT
+open Lwt
+
+(* IP Configuration, all you need besides dhcpd.conf. *)
+let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
```

Comment:
If there is no other way, I would vote for pushing as it is and then fixing
once functoria is released.

Btw: im out of town until monday and wont have access to my machines.

On Friday, 13 November 2015, Amir Chaudhry <notifications@github.com> wrote:

> In dhcp/unikernel.ml
> <https://github.com/mirage/mirage-skeleton/pull/105#discussion_r44775417>:
>
> > @@ -0,0 +1,85 @@
> > +open V1_LWT
> > +open Lwt
> > +
> > +(* IP Configuration, all you need besides dhcpd.conf. *)
> > +let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
>
> Does that mean we should wait until functoria is properly released before
> adding this?
> … <#15100ba75d116d75_>
> On 12 Nov 2015, at 17:12, Thomas Gazagnaire <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote: In
> dhcp/unikernel.ml: > @@ -0,0 +1,85 @@ > +open V1_LWT > +open Lwt > + >
> +(* IP Configuration, all you need besides dhcpd.conf. *) > +let ipaddr =
> Ipaddr.V4.of_string_exn "192.168.1.5" it'd be very easy to do with the new
> functoria, see
> https://github.com/Drup/mirage-skeleton/blob/functoria/hello/config.ml#L3-L5
> — Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mirage/mirage-skeleton/pull/105/files#r44775417>.
>